README.md

```diff
@@ -2,41 +2,60 @@
 
 # Calico Community
 
-Calico is an open source project with an active development and user community. We encourage you to connect and take part!
+Project Calico is an open source project that provides networking and network security solution for containers,
+virtual machines, and bare-metal workloads. The Project Calico Community is committed to fostering an open and welcoming
+environment, with several ways to engage with other users and developers. We encourage and welcome your participation in
+the community.
 
-## Community Meetings
-The Calico maintainers host a monthly online community meeting where everyone is welcome. 
+## Slack
+The Calico Users [Slack group](https://slack.projectcalico.org) is a great place to get community support or connect with
+other users and Calico developers. [![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org)
 
-* [Community Event Calendar](
-https://calendar.google.com/calendar/embed?src=tigera.io_uunmavdev5ndovf0hc4frtl0i0%40group.calendar.google.com) (google calendar)
-* [Zoom Meeting](https://zoom.us/j/270294702) (additional dial in details in calendar)
-* [Meeting Details](
-https://docs.google.com/document/d/1b6-ZS7UmRP_-XDq4XnlaKyb7lW9xbnzJ4mt5JK7ASHY/) (agenda, minutes, and recordings)
+## Community Meetings
+The Calico maintainers host a monthly online community meeting where everyone is welcome. If you have something you’d like to
+talk about, please add it to the upcoming meeting agenda.
 
-## Slack
+- [Community Event Calendar](https://calendar.google.com/calendar/embed?src=tigera.io_uunmavdev5ndovf0hc4frtl0i0%40group.calendar.google.com)
+  (google calendar)
+- [Zoom Meeting](https://zoom.us/j/270294702) (additional dial in details in calendar)
+- [Meeting Details](https://docs.google.com/document/d/1b6-ZS7UmRP_-XDq4XnlaKyb7lW9xbnzJ4mt5JK7ASHY/)
+  (agenda, minutes, and recordings)
 
-The Calico Users [slack group](https://slack.projectcalico.org) is a great place to get community support or connect with other users and Calico developers.
+## Discourse
 
-[![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org)
+Join the conversation in the [Project Calico Discource](https://discuss.projectcalico.org).
```

Comment:
"Join the conversation" is a bit vague. See this PR I put together last week for adding Discourse where I had a go at positioning it vs slack.  (My guess is that people will actually use slack and discourse for the same things, but will gravitate to one vs another based on personal preference.  I suggest I close my PR in favor of your PR after tweaking the description in this PR.